<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Pablo" name="tests.testng.SinDoubleTest" tests="1" failures="0" timestamp="15 дек 2017 17:18:11 GMT" time="0.017" errors="1">
  <testcase name="sinTest" time="0.017" classname="tests.testng.SinDoubleTest">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: sinTest([Parameter{index=0, type=java.lang.Double, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.Double, declaredAnnotations=[]}])
Arguments: [(java.lang.Integer)1,(java.lang.Integer)2]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: sinTest([Parameter{index=0, type=java.lang.Double, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.Double, declaredAnnotations=[]}])
Arguments: [(java.lang.Integer)1,(java.lang.Integer)2]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at runner.Main.main(Main.java:26)
]]>
    </error>
  </testcase> <!-- sinTest -->
</testsuite> <!-- tests.testng.SinDoubleTest -->
